package.json

```diff
@@ -52,7 +52,12 @@
     "argparse": "^1.0.3",
     "jscs": "^1.13.1",
     "rollup": "^0.36.3",
+    "rollup-plugin-commonjs": "^5.0.5",
+    "rollup-plugin-node-resolve": "^2.0.0",
     "rollup-watch": "^2.5.0",
     "uglify-js": "^2.6.0"
+  },
+  "dependencies": {
+    "omnitone": "^0.1.6"
```

Comment:
yeah let's lock in a version and then we can be sure to add this to our updates list